### ISSUE TITLE
agp 3.5c12 issue

### ISSUE THREAD

**Issue by adamfit** [mannodermaus/android-junit5#167]
My project won't gradle configure on this version (a particular library module is using the plugin). I couldn't figure out how to isolate the problem into a test project, but I got a fishy looking stacktrace so here we go.

agp: 3.5-alpha12
gradle: 5.4-rc1
android-junit5: 1.4.2.0

trace:
```
Caused by: groovy.lang.MissingPropertyException: No such property: javaOutputDir for class: com.android.build.gradle.internal.scope.VariantScopeImpl
        at de.mannodermaus.gradle.plugins.junit5.GroovyInterop.variantScope_javaOutputDirs(GroovyInterop.groovy:86)
        at de.mannodermaus.gradle.plugins.junit5.InteropKt.getSafeJavaOutputDirs(Interop.kt:22)
        at de.mannodermaus.gradle.plugins.junit5.providers.JavaDirectoryProvider.classFoldersOf(Java.kt:28)
        at de.mannodermaus.gradle.plugins.junit5.providers.JavaDirectoryProvider.mainClassDirectories(Java.kt:17)
        at de.mannodermaus.gradle.plugins.junit5.providers.BaseKt.mainClassDirectories(Base.kt:50)
        at de.mannodermaus.gradle.plugins.junit5.tasks.AndroidJUnit5JacocoReport$ConfigAction.execute(Jacoco.kt:106)
        at de.mannodermaus.gradle.plugins.junit5.tasks.AndroidJUnit5JacocoReport$Companion$create$1.execute(Jacoco.kt:37)
        at de.mannodermaus.gradle.plugins.junit5.tasks.AndroidJUnit5JacocoReport$Companion$create$1.execute(Jacoco.kt:30)
        at org.gradle.api.internal.tasks.DefaultTaskContainer.create(DefaultTaskContainer.java:360)
        at de.mannodermaus.gradle.plugins.junit5.tasks.AndroidJUnit5JacocoReport$Companion.create(Jacoco.kt:36)
        at de.mannodermaus.gradle.plugins.junit5.AndroidJUnitPlatformPlugin$configureJacocoTasks$1.execute(Plugin.kt:97)
        at de.mannodermaus.gradle.plugins.junit5.AndroidJUnitPlatformPlugin$configureJacocoTasks$1.execute(Plugin.kt:21)
        at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:158)
        at de.mannodermaus.gradle.plugins.junit5.AndroidJUnitPlatformPlugin.configureJacocoTasks(Plugin.kt:90)
        at de.mannodermaus.gradle.plugins.junit5.AndroidJUnitPlatformPlugin.access$configureJacocoTasks(Plugin.kt:21)
        at de.mannodermaus.gradle.plugins.junit5.AndroidJUnitPlatformPlugin$apply$3.execute(Plugin.kt:44)
        at de.mannodermaus.gradle.plugins.junit5.AndroidJUnitPlatformPlugin$apply$3.execute(Plugin.kt:21)
        at org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingAction$1$1.run(DefaultListenerBuildOperationDecorator.java:150)
```

**Comment by mannodermaus** [comment]
Thanks for reporting! Looks like yet another breakage because of the plugin having to reach into AGP internals due to missing official APIs. I haven't had a chance to read the latest versions' source code, but good to know that there's an action point in here.

**Comment by mannodermaus** [comment]
This will be fixed in the next version. If you need the fix now, you can grab the latest `1.4.2.1-SNAPSHOT`!

**Comment by adamfit** [comment]
Thanks!
